apps/ios/src/FluentUITester.xcodeproj/project.pbxproj

```diff
@@ -285,13 +285,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 9F3A29F4244F657400A401D2 /* Debug.xcconfig */;
 			buildSettings = {
+				TARGETED_DEVICE_FAMILY = "1,2";
```

Comment:
I'm gonna take a wild guess and say this should probably be moved to an xcconfig?

Reply:
Yes definitely!